5.5/root/usr/share/container-scripts/mysql/lib.sh

```diff
@@ -0,0 +1,3 @@
+function info-log {
+	echo "CONTAINER[${HOSTNAME:-}] `date +%T`     $@"
```

Comment:
I was thinking about using something more precise, but decided for simplicity because I can't imagine how it could be useful and this way it is nicer to look at. And keep in mind that this just happens until the final exec, so it is mostly useful when you want to know how long did the slave wait for master, or similar stuff. If you want to know the exact date and time when you first started the container, you can just ask Docker.